src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseCloseServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.CASE_CLOSED_POSITION;
+
+class CaseCloseServiceTest {
+
+    private CaseCloseService caseCloseService;
+    private CaseData caseData;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        caseCloseService = new CaseCloseService();
+        caseData = new CaseData();
+    }
+
+    @Test
+    void shouldValidateReinstateClosedCase_IsCaseClose_ReturnOne() {
+        String errorMessage = "This case cannot be reinstated with a "
```

Comment:
A better approach is to make use of the constant you've defined in the service. If you remove the private modifier here:
`private static final String REINSTATE_CANNOT_CASE_CLOSED_ERROR_MESSAGE`

then you can use the variable in this unit test

`assertEquals(REINSTATE_CANNOT_CASE_CLOSED_ERROR_MESSAGE, errors.get(0));`

Reply:
Thank you James. Updated.